store.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Karma store</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+
+  <body>
+
+    <header class="store-header">
+
+        <img class="svg-logo"
+            src="./img/karma-logo.svg" 
+            alt="Karma company logo">
+        </img>
+        <nav class="nav-bar">
+            <ul class="navigation__list">
+                <li class="active">Meet Karma</li> 
+                <li>How it Works </li>
+                <li><a href="./store.html">Store</a></li>
+                <li>Blog</li>
+                <li>Help</li>
+                <li>Login</li>
+            </ul>
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+                
+            </div>
+        </nav>              
+    </header>
+
+  <!-- <main> -->
+
+    <main class="store-main">
+      <section class="karma-order-form">
+        <h1 class>Order your Karma wifi device today!</h1>
+        
+        <form>
+          <div class="first-last-name">
+            <fieldset>
```

Comment:
nice use of `form` and `fieldset` elements

Reply:
Thank you,Ellie.